packages/core/src/util/normalizeConfig.js

```diff
@@ -718,10 +718,23 @@ function normalizeIgnoreEndpoints(config) {
       }
     });
   } else if (process.env.INSTANA_IGNORE_ENDPOINTS) {
-    // The environment variable name and its format are still under discussion.
-    // It is currently private and will not be documented or publicly shared.
+    // Expected format: "service:endpoint1,endpoint2;service2:endpoint1"
     try {
-      config.tracing.ignoreEndpoints = JSON.parse(process.env.INSTANA_IGNORE_ENDPOINTS);
+      const parsedEndpoints = Object.fromEntries(
+        process.env.INSTANA_IGNORE_ENDPOINTS.split(';')
```

Comment:
question: Do you support whitespaces?

e.g. "service:endpoint1,endpoint2; service2:endpoint1"

Reply:
Yes, updated the code.